tests/api/runtimes/test_kubejob.py

```diff
@@ -5,10 +5,17 @@
 import deepdiff
 import pytest
 from fastapi.testclient import TestClient
+from kubernetes import client as k8s_client
```

Comment:
Don't use `from X import Y` please
You can use `import X as Y` if you want

Reply:
I have updated pod.py, but in case we want all files to follow this convention I think a separate PR will have to deal with this requirement.